### DISCUSSION TITLE
using read only instance for read only transactions

### DISCUSSION THREAD

**Discussion by palharsh** [aws/aws-advanced-jdbc-wrapper#322] [Q&A] [answered]
Hi,
from the readme I see
> We are investigating Read/Write splitting for a future release.

Is this in the works already or is there an expected timeline for this?

Thanks!

**Comment by davecramer** [comment, marked as answer]
The code is already in the repository and has been released.

**Reply by palharsh** [reply]
thanks for the quick reply!
